README.md

```diff
@@ -33,3 +33,13 @@ The software will print an output to the console for each packet that is receive
 If the NCP fails to receive a valid frame with the timeout period set with the ```timeout``` command line parameter, then the NCP will be restarted. This will allow the sniffer to recover from serial port or NCP communications problems. The timer defaults to 30 seconds.
 
 A compiled JAR file can be found [here](https://www.cd-jackson.com/downloads/ZigBeeSniffer.jar) along with [further documentation](https://www.cd-jackson.com/downloads/ZigBeeWiresharkSniffer.pdf).
+
+When using Wireshark to display the packets, the raw IEEE 802.15.4 packet received by the Ember module is first encapsulated in a "TI CC24xx" frame format, then in a ZEPv2 (ZigBee Encapsulation Protocol version 2) frame format before being sent using UDP.
+Using the "TI CC24xx" frame format permit passing the RSSI value but has also limitations:
+* The "RSSI" value is correctly sent using a signed integer value in dBm
+* The "FCS Valid" field is always set to true as the Ember module discards invalid packets
+* The "LQI Correlation Value" is limited to a range of 0 to 127 (whereas the Ember module and the norm are defining this value for the range 0 to 255), so the displayed value is divided by 2.
+
+The real LQI value reported by the module in the range 0 to 255 should be displayed in the ZigBee Encapsulation Protocol section, but due to a bug, this isn't actually the case (see https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=16369).
+
+Due to a limitation in the new Ember modules based on the EFR32, the LQI value isn't based on the bit error rate of the demodulator but is related to the RSSI value (see https://www.silabs.com/community/wireless/zigbee-and-thread/knowledge-base.entry.html/2017/08/15/lqi_in_silicon_labs-vvSq)
```

Comment:
I'm not sure I would put this in the doc. Different devices generate the LQI through different means - in any case the LQI that is shown is the LQI from the device. If the user uses a different device, or if Silabs change the way it's generated, then this will be wrong and IMHO this is simply a detail of the inner working of the chip.

Reply:
All the new modules produced by Silabs are based on the EFR32. I was thinking it was interessing to explain this limitation. As there is no standard rule to calculate the LQI, this can removed if you think it may be misleading.

Reply:
Ok, how about we leave the link, but remove the description. The link covers the EFR32, but also the EM35x, and also may be updated if future versions of the EFR32 use a different algorithm. I would just change this to something like "For information on how the LQI is calculated in Silabs chips, refer to _link_"

Reply:
This is fine for me, I'll see with @mikomarrache how to change the wording in the commit.

Reply:
@cdjackson  I changed the wording.